projects/keycloak/secure-app-realm.json

```diff
@@ -0,0 +1,1579 @@
+{
```

Comment:
I think it might be more maintainable if we manually write the realm.json file. Its around 95% smaller in size and makes updating out realm config and debugging easier. I wrote a simplified real config here - https://gist.github.com/TommyJ1994/7eec79f8e6721e6037a7218e2b3f619b